CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ We used to have incremental versioning before `0.1.0`.
 ### Features
 
 - Forbids using `Literal[None]` in function annotations
+- Fixes 
```

Comment:
Looks unfinished 🙂 And it should be located in `### Bugfixes` section

Reply:
Oops, my bad, untimely Ctrl-Z. :D Will fix.